package.json

```diff
@@ -84,5 +85,8 @@
       "last 1 safari version"
     ]
   },
-  "homepage": "."
+  "homepage": ".",
+  "devDependencies": {
+    "@types/lodash": "^4.14.169"
```

Comment:
Should this be a dev dependency? I thought we discussed with @stuart-madetech that all dependencies can go in normal `dependencies` key

Reply:
Yes they do, totally right (because Create React App is statically generated so _every_ dependency is a dev dependency)